spec/system/sign_in_redirect_spec.rb

```diff
@@ -0,0 +1,118 @@
+require "rails_helper"
+
+RSpec.describe "Sign in redirect", service: :placements, type: :system do
+  before do
+    user_exists_in_dfe_sign_in(user:)
+  end
+
+  context "when I am a user" do
+    let(:user) { create(:placements_user) }
+    let(:school) { create(:placements_school, users: [user]) }
+
+    scenario "Logging in via the root path redirects to the placements page" do
```

Comment:
Should this be "organisations" rather than "placements"?

```suggestion
    scenario "Logging in via the root path redirects to the organisations page" do
```

Reply:
And from what I can tell, it looks like the user goes to the "organisations" page rather than "placements" because they have two organisations.

1. Is auto-assigned by the factory when we call `create(:placements_user)`
2. The other is assigned when we add the user to the school `create(:placements_school, users: [user])`

So the user always has two organisations, hence they see the "organisations" index page.

Reply:
You're correct on both counts, this was from an earlier implementation that I ended up changing :)